docs/src/main/asciidoc/openapi-swaggerui.adoc

```diff
@@ -260,6 +260,16 @@ quarkus.smallrye-openapi.path=/swagger
 ----
 ====
 
++[NOTE]
++====
++You can request the OpenAPI in JSON format using the `format` query parameter. For example:
++[source, properties]
++----
++/q/openapi?format=json
++----
++====
++
+
```

Comment:
Not sure why you added `+` at the beginning of each line?
```suggestion
[NOTE]
====
You can request the OpenAPI in JSON format using the `format` query parameter. For example:
[source, properties]
----
/q/openapi?format=json
----
====

```

Reply:
Oh wow, copy paste error. I originally cloned and changed the "guides" repo and copied the changes  🤦 

Fixed now.